dbms/programs/odbc-bridge/getIdentifierQuote.cpp

```diff
@@ -0,0 +1,45 @@
+#include "getIdentifierQuote.h"
+#if USE_POCO_SQLODBC || USE_POCO_DATAODBC
+
+#if USE_POCO_SQLODBC
+#include <Poco/SQL/ODBC/ODBCException.h>
+#include <Poco/SQL/ODBC/SessionImpl.h>
+#include <Poco/SQL/ODBC/Utility.h>
+#define POCO_SQL_ODBC_CLASS Poco::SQL::ODBC
+#endif
+#if USE_POCO_DATAODBC
+#include <Poco/Data/ODBC/ODBCException.h>
+#include <Poco/Data/ODBC/SessionImpl.h>
+#include <Poco/Data/ODBC/Utility.h>
+#define POCO_SQL_ODBC_CLASS Poco::Data::ODBC
+#endif
+
+
+namespace DB
+{
+    std::string getIdentifierQuote(SQLHDBC hdbc)
+    {
+        std::string identifier = "";
+
+        SQLSMALLINT t;
+        SQLRETURN r = POCO_SQL_ODBC_CLASS::SQLGetInfo(hdbc, SQL_IDENTIFIER_QUOTE_CHAR, NULL, 0, &t);
```

Comment:
Actually we know that the size of SQL_IDENTIFIER_QUOTE_CHAR is one or zero bytes. So maybe here we can allocate buffer (identifier) and request ODBC only once?

Reply:
I propose to keep it consistent with bloody ODBC-interface, and not make too tricky and fragile.